Use only Daml Script in Explicit Disclosure docs example (#594)
digital-asset/docs.daml.com: docs/3.0.0/docs/app-dev/explicit-contract-disclosure.rst

so **Buyer** must provide them as disclosed contracts in the command submission exercising ``Offer_Accept``. To
do so, the contracts' stakeholders must fetch them from the ledger and make them available to the **Buyer**.

::

       disclosedStock <- fromSome <$> queryDisclosure stockExchange stockCid
       disclosedOffer <- fromSome <$> queryDisclosure seller offerCid
       disclosedPriceQuotation <- fromSome <$> queryDisclosure stockExchange priceQuotationCid

> Then, the **Buyer** attaches the disclosed contract payloads to the command submission in which he accepts the offer.

Reviewer: ```suggestion
Then, the **Buyer** attaches the disclosed contract payloads to the command submission that accepts the offer.
```

::

>        -- -- Submission using disclosed contracts

Author: Good point about the comment, let's delete it.
